template.json

```
@@ -928,12 +928,22 @@
         "toughness": false,
         "willpower": false
       },
+      "hasCost": false,
       "increase": 0,
       "isBonus": false,
       "isSeperate": false,
       "isSpendable": false,
       "level20Value": 0,
       "levels": [],
+      "powerCost": 0,
+      "rolePoints": {
+        "max": 0,
+        "value" : 0,
+        "secondary": {
+          "max": 0,
+          "value" : 0
+        }
+      },
```

```suggestion
      "points": {
        "primary": {
          "max": 0,
          "value" : 0
        },
        "secondary": {
          "max": 0,
          "value" : 0
        }
      },
```
Thoughts on this structure? Also I'm not big on role points having role points, so I just changed it to `points`.

---

I like the structure have to do a deep dive to make sure this is changed everywhere.